Commands/compute-diagnostic/spot-placement-recommender/_generate.md

```diff
@@ -1,16 +0,0 @@
-# [Command] _compute-diagnostic spot-placement-recommender generate_
-
-Generates placement scores for Spot VM skus.
-
-## Versions
-
-### [2021-06-01-preview](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.xml) **Stable**
```

Comment:
Please also remove the following two files as well [2021-06-01-preview.json](https://github.com/Azure/aaz/tree/main/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.xml) and [2021-06-01-preview.xml](https://github.com/Azure/aaz/tree/main/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb21wdXRlL2xvY2F0aW9ucy97fS9kaWFnbm9zdGljcy9zcG90cGxhY2VtZW50cmVjb21tZW5kZXIvZ2VuZXJhdGU=/2021-06-01-preview.json)

Reply:
thanks. removed